backend/clickhouse/log_row.go

```diff
@@ -72,9 +72,9 @@ func (l *LogRow) Cursor() string {
 
 type LogRowOption func(*LogRow)
 
-func WithLogAttributes(resourceAttributes, eventAttributes map[string]any, isFrontendLog bool) LogRowOption {
+func WithLogAttributes(resourceAttributes, spanAttributes, eventAttributes map[string]any, isFrontendLog bool) LogRowOption {
```

Comment:
As a thought, since we really don't make a distinction between the attributes, we might just want to make a single param `attributes` and force the callers to pass one map in.

Reply:
i guess there could be another method that brings the maps together before they are passed here, but i think i'd rather keep them as separate arguments in case we want to nest them under different keys in the future, or do different processing depending on the attribute source.